lua/fpp/server/ownability.lua

```diff
@@ -286,13 +286,16 @@ end
 Networking
 ---------------------------------------------------------------------------]]
 util.AddNetworkString("FPP_TouchabilityData")
--- Sends 13 + 8 + 5 + 20 = 46 bits of ownership data per entity
+
+-- Sends MAX_EDICT_BITS(13) + (1, 8) + 5 + 20 = (39, 46) bits of ownership data per entity
+local MAX_PLAYER_BITS = math.ceil(math.log(1 + game.MaxPlayers()) / math.log(2))
+
 local function netWriteEntData(ply, ent)
     -- EntIndex for when it's out of the PVS of the player
-    net.WriteUInt(ent:EntIndex(), 13)
+    net.WriteUInt(ent:EntIndex(), MAX_EDICT_BITS)
 
     local owner = ent:CPPIGetOwner()
-    net.WriteUInt(IsValid(owner) and owner:EntIndex() or 255, 8)
+    net.WriteUInt(IsValid(owner) and owner:EntIndex() or 0, MAX_PLAYER_BITS)
```

Comment:
This changes a `255` to `0`. IIRC `0` refers to the world entity. This could make the client unable to differentiate between a _genuine_ world owned entity, and an otherwise invalid player. This can cause trouble when referring to a disconnected player. 

I see that the client is also changed to interpret `0` as `-1`. I also see that `255` cannot be used, because of the `MAX_PLAYER_BITS` change.

What are your thoughts on this?

Reply:
Looks like it is, thanks!